core/common/src/main/java/alluxio/conf/ReconfigurableRegistry.java

```diff
@@ -45,12 +47,24 @@ public static synchronized boolean unregister(Reconfigurable listener) {
    * @return false if no listener related to the given property, otherwise, return false
    */
   public static synchronized boolean update() {
-    for (Reconfigurable listener : new LinkedList<>(LISTENER_LIST)) {
+    for (Reconfigurable listener : new ArrayList<>(LISTENER_LIST)) {
```

Comment:
BTW, Why do we need to copy LISTENER_LIST here?

Reply:
good call, it's synchronized so there's no need for a copy

Reply:
Good catch